internal/util/cluster_mapping_test.go

```diff
@@ -138,14 +138,14 @@ func TestGetClusterMappingInfo(t *testing.T) {
 		currentTT := tt
 		t.Run(currentTT.name, func(t *testing.T) {
 			t.Parallel()
-			clusterMappingConfigFile = fmt.Sprintf("%s/mapping-%d.json", mappingBasePath, currentI)
+			mappingConfigFile := fmt.Sprintf("%s/mapping-%d.json", mappingBasePath, currentI)
 			if len(currentTT.mappingFilecontent) != 0 {
-				err = ioutil.WriteFile(clusterMappingConfigFile, currentTT.mappingFilecontent, 0o600)
+				err = ioutil.WriteFile(mappingConfigFile, currentTT.mappingFilecontent, 0o600)
```

Comment:
function name in the error message needs to be changed at lines 145 and 150

Reply:
Don't we have anything like __FILE__, __LINE__, __func__ in go?

Reply:
I don't think those kind of variables exist in go. It might be possible to obtain them, but its more work than that it is useful here.